spec/tree/values/nodes/ParagraphSpec.js

```diff
@@ -1,43 +1,30 @@
-import TextContainer from "../../../../src/tree/values/nodes/TextContainer";
-import FormattingElement from "../../../../src/tree/values/FormattingElement";
 import Paragraph from "../../../../src/tree/values/nodes/Paragraph";
 
 describe( "Paragraph tree node", () => {
 	describe( "constructor", () => {
-		it( "can make a new Paragraph tree node with a paragraph tag", () => {
-			const formattingElements = [
-				new FormattingElement( "a", 25, 29 ),
-			];
-			const text = "This is some text with a link.";
-			const textElement = new TextContainer( text, formattingElements );
-
-			const startIndex = 0;
-			const endIndex = 30;
-			const tag = "p";
-
-			const paragraphElement = new Paragraph( textElement, startIndex, endIndex, tag );
-
-			expect( paragraphElement.textContainer ).toEqual( textElement );
-			expect( paragraphElement.startIndex ).toEqual( 0 );
-			expect( paragraphElement.endIndex ).toEqual( 30 );
+		test( "can make a new Paragraph tree node", () => {
```

Comment:
Everywhere else we use `it`, we should adhere to that.

Also then this text needs to be changed to fit that. So `it( "makes a new Paragraph tree node"`. But I think `creates` sounds better here.

```suggestion
		it( "creates a new Paragraph tree node", () => {
```